doc/docs/Python_Tutorials/Adjoint_Solver.md

```diff
@@ -35,75 +34,86 @@ Python using [autograd](https://github.com/HIPS/autograd) as well as
 The adjoint solver supports inverse design and [topology
 optimization](https://en.wikipedia.org/wiki/Topology_optimization) by
 providing the functionality to wrap an optimization library around the
-gradient computation. This is demonstrated in several tutorials below.
+gradient computation. The adjoint solver also supports enforcing
+minimum feature sizes on the optimal design in 1D (line width and
+spacing) and 2D (circular holes and islands). This is demonstrated in
```

Comment:
note that the 2D constraints are for arbitrary "enclosed shapes" (e.g. not restricted to circular).

Reply:
Fixed.